.changeset/whole-candles-ask.md

```diff
@@ -0,0 +1,5 @@
+---
+'@posthog/ai': patch
```

Comment:
this is probably more than a patch, i would do minor bump

Reply:
I didn't add any new features, they were bug fixes and no changes in the public API, that's why I thought it would be best put as a patch, WDYT?